torch_frame/gbdt/tuned_xgboost.py

```diff
@@ -174,10 +177,15 @@ def objective(
 
         boost = xgboost.train(self.params, dtrain,
                               num_boost_round=num_boost_round,
-                              early_stopping_rounds=50, verbose_eval=False,
-                              evals=[(dvalid, 'validation')],
-                              callbacks=[pruning_callback])
-        pred = boost.predict(dvalid)
+                              early_stopping_rounds=early_stopping_rounds,
+                              verbose_eval=False, evals=[
+                                  (dvalid, 'validation')
+                              ], callbacks=[pruning_callback])
+        if boost.best_iteration:
```

Comment:
Seems we missed this... https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.Booster.predict